hack/lib/version.sh

```diff
@@ -26,6 +26,7 @@
 #    KUBE_GIT_VERSION - "vX.Y" used to indicate the last release version.
 #    KUBE_GIT_MAJOR - The major part of the version
 #    KUBE_GIT_MINOR - The minor component of the version
+#    KUSTOMIZE_VERSION - The semantic version of kustomize embedded in kubectl
```

Comment:
I'd really like to avoid this plumbing here, since the compiled in module info is in reach with go 1.18 and this build script services many binaries, not just kubectl